js/client-chat.js

```diff
@@ -747,6 +778,14 @@
 					this.add('/challenge - Open a prompt to challenge a user to a battle.');
 					this.add('/challenge [user] - Challenge the user [user] to a battle.');
 					return false;
+				case 'accept':
+					this.add('/accept - Accept a challenge if only one is pending.');
+					this.add('/accept [user] - Accept a challenge from the specified user.');
```

Comment:
Oh. It might help if ``/accept [user]`` actually worked. I really need to get better with testing